package.json

```diff
@@ -24,22 +24,21 @@
   },
   "homepage": "https://github.com/climatiq/eslint-config-climatiq#readme",
   "peerDependencies": {
+    "@next/eslint-plugin-next": "^13.4.6",
     "@typescript-eslint/eslint-plugin": ">=5.3.1",
     "@typescript-eslint/parser": ">=5.3.1",
     "eslint-config-prettier": ">=8.3.0",
     "eslint-plugin-prettier": ">=4.0.0",
-    "prettier": ">=2.4.1"
```

Comment:
Do we not need prettier at all in either `peerDependencies` or `dependencies`?

Reply:
@jcyling Sorry, forgot to push my last change